docs/installation/frontend-development.md

```diff
@@ -0,0 +1,111 @@
+# Installation for development
+
+<div class="Alert Alert--warning">
+
+1. First visit [backend installation][be] part to make the application work properly<br>
+2. Use backend URL in your `.env` file or use [cli commend](installation/frontend?id=configuration).
+
+</div>
+
+## Requirements
+---
+
+1. In case you don't have [node.js][node] installed - install the latest stable version.
```

Comment:
I guess docker repository can be used for development as well

Reply:
docker instllation is in a different place.